src/index.js

```diff
@@ -31,9 +31,13 @@ export default class LazilyRender extends React.Component<LazilyRenderProps, Laz
     hasBeenScrolledIntoView: false
   };
 
+  isBackCompatMode() {
+    return document.compatMode === "BackCompat";
```

Comment:
I think if we move this into a separate file like **utils/isBackCompactMode** will make the testing later on easier, since we can just do: 

```
jest.mock('../utils/isBackCompatMode')

import {isBackCompatMode} from '../utils/isBackCompatMode';

isBackCompatMode.mockReturnValue(true|false)...
```